src/index.tsx

```diff
@@ -134,7 +134,8 @@ const DateBlock: React.FC<DateBlockProps> = ({
     }, [scrollRef.current])
 
     const handleChange = ({nativeEvent}: any) => {
-        const digit = nativeEvent.contentOffset.y / dHeight + digits[0];
+        let digit = nativeEvent.contentOffset.y / dHeight + digits[0];
+        digit = parseInt(digit.toFixed(0), 10);
 
```

Comment:
Please describe in more detail what the problem was. The issue you link to doesn't describe the problem enough.

Reply:
the current problem is that the date is not updated because currently in the library the verification is done on an integer while the application never returns an integer number

The solution and therefore cast the figure as an integer

Reply:
Thanks